<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="map_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>551</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>시군구</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>행정동</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>연령</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>남</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>여</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>데이터기준일</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>광주광역시 북구 1인 가구 현황</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
